### PR TITLE
add line in readme

### PR DESCRIPTION
**Pull request by willkoo**
Added a title in readme file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,3 @@
 Rails app generated with [lewagon/rails-templates](https://github.com/lewagon/rails-templates), created by the [Le Wagon coding bootcamp](https://www.lewagon.com) team.
+
+JPL-app readme
```
Comment (collich) [inline README.md]: JPL good

File: README.md [hunk 2]
```diff
@@ -1 +1,3 @@
 Rails app generated with [lewagon/rails-templates](https://github.com/lewagon/rails-templates), created by the [Le Wagon coding bootcamp](https://www.lewagon.com) team.
+
```
Comment (wyxlester) [inline README.md:2]: naughty boy

**Review by collich — DISMISSED** [review]

File: README.md [hunk 3]
```diff
@@ -1 +1,3 @@
 Rails app generated with [lewagon/rails-templates](https://github.com/lewagon/rails-templates), created by the [Le Wagon coding bootcamp](https://www.lewagon.com) team.
```
Comment (wyxlester) [inline README.md:1]: not good

**Review by wyxlester — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (willkoo) [reply README.md:1]: where not good?

**Review by wyxlester — APPROVED** [review]

**Pull request merged**